src/mono/wasm/runtime-test.js

```diff
@@ -189,6 +189,7 @@ var Module = {
 						{
 							var fullPath = directory != '/' ? directory + '/' + files[j] : files[j];
 							var content = new Uint8Array (read ("supportFiles/" + fullPath, 'binary'));
+							console.log(fullPath);
```

Comment:
Do we need to log the fullPath or was this for testing purposes? I remember this making the output larger.

Reply:
This was for debugging purposes. I will take that out.